tests/units/transformers/test_relevant_feature_augmenter.py

```diff
@@ -72,11 +76,14 @@ def test_evaluate_only_added_features_true(self):
         augmenter.fit(X, y)
         transformed_X = augmenter.transform(X.copy())
 
+        fit_transformed_X = augmenter.fit_transform(X, y)
+
         self.assertEqual(sum(["pre_feature" == column for column in transformed_X.columns]), 1)
+        self.assertEqual(sum(["pre_feature" == column for column in fit_transformed_X.columns]), 1)
 
-    def test_evaluate_only_added_features_false(self):
+    def filter_only_tsfresh_features_false(self):
```

Comment:
Is there a reason you removed the `test_` prefix from this function? This means it will not be picked up by most of the unittest frameworks

Reply:
deleted accidently. will be fixed